### PR TITLE
PR for Filter functionality

### PR DESCRIPTION
**Pull request by manismk**
- Added Filter functionality
- [Link](https://deploy-preview-14--key-notes.netlify.app/)

### CODE REVIEW

File: src/components/filters/Filters.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+          }
+        >
+          <option value="newest">Newest On Top</option>
+          <option value="oldest">Oldest On Top</option>
+        </select>
+      </div>
+      <div className="priority--container">
+        <p>Priority</p>
+        {priorityData.map((priority) => (
+          <span key={priority.id}>
```
Comment (shraddha-amber) [inline src/components/filters/Filters.jsx:45]: could destructure properties of priority

File: src/context/filter-context.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  const [filterState, filterDispatch] = useReducer(filterReducer, {
+    filteredNotes: notes,
+    sort: "newest",
+    priority: "all",
+  });
+
+  useEffect(() => {
+    filterDispatch({
+      type: filterActions.NOTES_CHANGE,
+      payload: { notes: notes },
```
Comment (shraddha-amber) [inline src/context/filter-context.js:19]: can just pass notes since key and value are the same

File: src/utils/getSortedNotes.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+export const getSortedNotes = (notes, sort) => {
+  if (sort === "newest")
```
Comment (shraddha-amber) [inline src/utils/getSortedNotes.js:2]: could make newest and oldest as constant values to avoid typo

**Review by shraddha-amber — APPROVED** [review]
* Functionality works properly
* Clean and well structured code

**Review by JowelTisso — APPROVED** [review]
* Nice constants variables
* Good folder structure
* Clean code

**Pull request merged**